### PR TITLE
Add Capital gain tax calculator

### PR DESCRIPTION
**Pull request by acsehi**

### CODE REVIEW

File: capgain.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# http://cgtcalculator.com/
+import csv
+import datetime
+import urllib.request
+import json
+
+
+def ex(date):
+    url = 'https://api.exchangerate.host/'+date+'?base=USD&symbols=GBP'
```
Comment (kovacsbalu) [inline capgain.py:9]: format raw date here:
```
def ex(raw_date):
    formatted_dt = datetime.datetime.strptime(raw_date, '%b-%d-%Y').strftime('%Y-%m-%d')
    url = f"https://api.exchangerate.host/{formatted_dt}?base=USD&symbols=GBP"
```

File: capgain.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    with urllib.request.urlopen(url) as f:
+        print(url + " downloaded")
+        r = json.loads(f.read().decode('utf-8'))
+        return r["rates"]["GBP"]
+
+
```
Comment (kovacsbalu) [inline capgain.py:15]: new date formatter func for L24,41,51

File: capgain.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            date_close.append(dc)
+            quantity_close.append(row[1])
+            cost = float(row[3])/float(row[1])
+            cost_close.append(cost*float(exchange_rate_sell))
+
+with open('cgt.tsv', 'w', newline='',) as csvfile:
+    fieldnames = ['Action', 'Date', 'Stock_Name',
+                  'Quantity', 'price', 'brokercost', 'tax']
+    writer = csv.DictWriter(csvfile, fieldnames=fieldnames, delimiter='\t')
+    for t in range(0, len(date_open)):
```
Comment (kovacsbalu) [inline capgain.py:63]: use zip or enumerate

**Review by kovacsbalu — CHANGES_REQUESTED** [review]

**Comment by acsehi** [comment]
Comments will be fixed in new PR

**Pull request merged**